vflow/ipfix_windows.go

```diff
@@ -0,0 +1,26 @@
+//: ----------------------------------------------------------------------------
+//: Copyright (C) 2017 Verizon.  All Rights Reserved.
+//: All Rights Reserved
+//:
+//: file:    ipfix.go
+//: author:  Mehrdad Arshad Rad - copied by Jeremy Rossi, but not important
+//: date:    02/01/2017
+//:
+//: Licensed under the Apache License, Version 2.0 (the "License");
+//: you may not use this file except in compliance with the License.
+//: You may obtain a copy of the License at
+//:
+//:     http://www.apache.org/licenses/LICENSE-2.0
+//:
+//: Unless required by applicable law or agreed to in writing, software
+//: distributed under the License is distributed on an "AS IS" BASIS,
+//: WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//: See the License for the specific language governing permissions and
+//: limitations under the License.
+//: ----------------------------------------------------------------------------
+
```

Comment:
Suggesting that we should `// +build windows` in this file.  That works, but in the case it's also that way based on the file name as _windows.go does the same things.